### PR TITLE
[SW-288] Fix top navigation bar

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

File: src/popup/components/Header.vue [hunk 1]
```diff
@@ -3,66 +3,68 @@
...
+        >
+          <Logo />
+        </Component>
+        <BtnPlain
+          v-if="showBack"
+          class="icon-btn"
+          @click="back"
+        >
+          <Back data-cy="back-arrow" />
+        </BtnPlain>
```
Comment (pstachel) [inline src/popup/components/Header.vue:25]: You should use BtnIcon for it

File: src/popup/components/Header.vue [hunk 2]
```diff
@@ -3,66 +3,68 @@
...
-
-      <RouterLink
-        v-if="$route.path !== '/more' && !$route.meta.closeButton"
-        class="icon-btn"
-        to="/more"
-        data-cy="page-more"
+        <Truncate
+          :str="pageTitle ?
+            pageTitle :
+            title ? $t(`pages.titles.${title}`) : ''"
```
Comment (pstachel) [inline src/popup/components/Header.vue:34]: I'd suggest moving it into computed

**Review by pstachel — CHANGES_REQUESTED** [review]

File: src/icons/logo-header.svg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<svg width="33" height="32" viewBox="0 0 33 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (peronczyk) [inline src/icons/logo-header.svg:1]: After some testing I think adding new logo file makes new sensy as the old one was OK. The size difference could be fixed by changing CSS.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/components/Header.vue (hunk 1 above)
Comment (konradodo) [reply src/popup/components/Header.vue]: @peronczyk so i leave it now as agreed.

File: src/popup/components/Header.vue [hunk 4]
```diff
@@ -222,8 +226,6 @@ export default {
     }
 
     svg {
```
Comment (peronczyk) [inline src/popup/components/Header.vue:228]: This styles can be moved to `.home-icon`.

**Review by peronczyk — CHANGES_REQUESTED** [review]

**Review by pstachel — APPROVED** [review]

**Review by peronczyk — APPROVED** [review]

**Pull request merged**